libethereum/ExtVM.cpp

```diff
@@ -134,3 +134,22 @@ void ExtVM::suicide(Address _a)
 	m_s.subBalance(myAddress, m_s.balance(myAddress));
 	ExtVMFace::suicide(_a);
 }
+
+h256 ExtVM::blockHash(u256 _number)
+{
+	if (envInfo().number() < m_sealEngine.chainParams().u256Param("metropolisForkBlock") + 256)
```

Comment:
I think it was agreed last time that the opcode remains limited to the last 256 blocks.